book/listings/actions/7/window/imp.rs

```diff
@@ -45,7 +45,7 @@ impl ObjectImpl for Window {
         self.parent_constructed();
 
         // Setup
-        let instance = self.instance();
+        let instance = self.obj();
```

Comment:
Could you please rename this and similar cases to obj?

Reply:
Sure, makes sense. Done!